configs/AM62AX/AM62AX_linux_config.txt

```diff
@@ -40,6 +40,5 @@ Configuration Values
 'CONFIG_image_type'       : 'edgeai'
 'CONFIG_icss_support'     : 'yes'
 'CONFIG_rt_linux_support' : 'yes'
-'CONFIG_gpu_ip'           : 'Rogue_AXE'
```

Comment:
This is a good catch. Seems like different platform configs address lack of GPU differently. I didn't notice that until now. I'm going update those to this standard later.